onnxruntime_extensions/tools/pre_post_processing/docs/pre_post_processing/utils.md

```diff
@@ -57,4 +57,12 @@ Classes
     :
 
     `producer_idx: int`
+    :
+
+`Settings()`
+:   
+
+    ### Class variables
+
+    `pre_post_processing_onnx_opset`
```

Comment:
with the PrePostProcessor ctor taking an opset parameter now, maybe we don't need to expose this variable to users. and if my other comment sounds ok, perhaps we don't need it at all.